doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_flpr.rst

```diff
@@ -46,6 +46,12 @@ The primary purpose of this snippet is to enable the transfer of the FLPR code t
 When building for the ``nrf54h20dk/nrf54h20/cpuflpr`` target, a minimal sample is automatically loaded onto the application core.
 For more details, see :ref:`building_nrf54h_app_flpr_core`.
 
+Peripherals emulation on FLPR
```

Comment:
Note: Once this is merged, will look into merging both articles for L15 and H20 and moving them to the general section